composer.json

```diff
@@ -9,7 +9,7 @@
     "authors": [
```

Comment:
I think this entire section needs to be either rebranded or dropped (as mentioned somewhere internally today)